vizro-core/docs/pages/user-guides/card-button.md

```diff
@@ -411,7 +428,7 @@ Use the following pre-defined URL hashes in your image path to apply Vizro's def
             components=[
                 vm.Card(
                     text="""
-                    ![](assets/images/icons/hypotheses.svg#icon-top)
+                    ![](https://raw.githubusercontent.com/mckinsey/vizro/d24a6f0d4efdf3c47392458e64b190fa1f92b2a7/vizro-core/docs/assets/images/hypotheses.svg#icon-top)
```

Comment:
Does that link exist?

Reply:
Yes! Take a look https://py.cafe/snippet/vizro/v1#c=H4sIALiEvGYAA41STW_bMAz9K5p6SQDHTtK0XQP0sg9gAzZgGLbukOQgW3RMTJZciU6WFfnvo-yka7sVrXywKD6-x69bWTgNci6xbpwnscHf3qU1v5kgVBCbemlL7-reIQ6o62gs7dI2ag3iilHpF74NllbwISQDV0v5VnkttkiVwMLZpUx6d-GYxIKlcLXoX-Jhiog_UBwPwS9ipu574Hi1WA0qoibMs8yrbbpmmTZvA3iWIiZPWSari59oA-yyLvtMT2fqvBzrGZS6PC1mF6eX09nZazif5ZPLcakm5eU0n6qLHj4qnIdMuyJkKgSgkGHNRYas2jWOKggQ0rBZn8TiRuSaYWzI_RxPTk5E14NvsSGPvZ-YvRbYhLYW2hnnRUASqgZKuEWcdUFArRdKY4OhQLsWYJBS8Z1E6dlEY5TQWBATGFW0QcAa7vnSh3qfFRRgeaRoxUZ5dBxgMa8ScdMiDxp4JIr4ZtsYLD5yG9fgBVMazIGH3xryWEBIRCjAgMdw0wIHclYigH6k10_tMPN4hof7iv9dr7QKVe5ig7oNenc0B3GreDnib8XIbtsGwzRv0ejBXdQw9a0dDGUiPXAJ3M24U7zK3fSWlh20a-Judw9sqqa5RtjKOfkWEgka6b1VuWFMqUzgp2ZHlbMc0uycRg2jzTidnqdjDjZq51qS81u5AR8woqYs7Rx9dUx5exTzbCWyqDhXDwxa3HlI5bxG7NyipkrOJ2fjRNZof_TmaW99AFxXrBNN1BxWooE3zMq7_ZZ3W6EF_4RChI7yHsuQRkVeKferffJvFk-k-DeO25U2u4fRL6ju-YoOGT5XzbGQ_6dfHVgnF2cv0STw_KzMc6JHXFSN3z7pusp7tVjt_wD1JkFGLgUAAA